_posts/2024-07-22-emmy-farewell.md

```diff
@@ -34,15 +34,15 @@ The OLS team had been **collaborating with Emmy since 2019** through the Mozilla
 
 As the eLife cohort concluded and Emmy moved to her next role, it was a **natural progression for her to join the OLS team** to continue working with grassroots communities in open science through the cohort-based program, Open Seeds.
 
-## Scaling OLS' Finance and Operations
+## Scaling OLS's Finance and Operations
 
 With other OLS directors, Emmy was involved in scoping the [CZI grant]({% link _posts/2022-01-27-runway-to-sustainability-czi-funding.md %}) and responsible to manage the finances as we learned how to scale sustainably.
```

Comment:
```suggestion
 With other OLS directors, Emmy was involved in scoping the [CZI grant]({% link _posts/2022-01-27-runway-to-sustainability-czi-funding.md %}) and responsible for managing the finances as we learned how to scale sustainably.
```

Small suggestion, @malvikasharan. Feel free to ignore if you think it doesn't make a huge difference.